tools/backups2datalad/asyncer.py

```diff
@@ -427,6 +427,11 @@ async def async_assets(
                         await anyio.to_thread.run_sync(
                             partial(clone, source=src, path=ds.pathobj / asset_path)
                         )
+                        if config.zarr_gh_org is not None:
+                            await anyio.run_process(
```

Comment:
for my education in async/anyio: there is no need to somehow explicitly wait for the `partial(clone` above since it is 

a. `await`ed
b. `.run_sync`
c. both a. and b.

?

Reply:
The `partial(clone, ...)` is a synchronous function (it's `datalad.api.clone()`), so it's not awaited normally, but `anyio.to_thread.run_sync()` makes it run concurrently in a thread (because cloning can take a while), and the return value of `run_sync()` is an awaitable that, when awaited, suspends the calling coroutine until the clone-in-thread is done.

Reply:
thanks